src/test/regress/sql/multi_insert_select_window.sql

```diff
@@ -0,0 +1,594 @@
+--
+-- multi subquery window function covers the tests
```

Comment:
Need to update the comment, it is copy&paste from the other test.

And also could you make sure that we have the same format for these type of comments that appear in the beginning of the tests? An example from `multi_router_planner.sql`, but we have plenty.

(Please follow the same syntax for the other regression test file added in this PR)

```

-- ===================================================================
-- test router planner functionality for single shard select queries
-- ===================================================================
```